spec/controllers/all_casa_admins/casa_admins_controller_spec.rb

```diff
@@ -0,0 +1,170 @@
+require "rails_helper"
+
+RSpec.describe AllCasaAdmins::CasaAdminsController, type: :controller do
+  let(:all_casa_admin) { build(:all_casa_admin) }
+  let(:casa_org) { create(:casa_org) }
+
+  before do
+    all_casa_admin.update_attribute(:id, 5)
```

Comment:
The id of the all_casa_admin instances are nil, so I had to give them a number value so the sign in process would work.

Reply:
This seems very weird, that shouldn't happen...